<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PropertiesDashboardClass</class>
 <widget class="QWidget" name="PropertiesDashboardClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>801</width>
    <height>741</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PropertiesDashboard</string>
  </property>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>741</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget
{
background-color:#CCD0CD;

}</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>181</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
font-size:34px;
font-weight:bold;
color:#2B2A2A;
}</string>
    </property>
    <property name="text">
     <string>Properties</string>
    </property>
   </widget>
   <widget class="Line" name="line_8">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>731</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>761</width>
      <height>621</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarPolicy::ScrollBarAsNeeded</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::SizeAdjustPolicy::AdjustToContents</enum>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>759</width>
       <height>619</height>
      </rect>
     </property>
     <widget class="QLabel" name="label_38">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>460</y>
        <width>321</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
font-size:30px;
font-family:sans-serif;
font-weight:600;
}</string>
      </property>
      <property name="text">
       <string>No Properties To Show</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_39">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>500</y>
        <width>181</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
font-family:sans-serif;
color:#808080;
font-weight:600;
}</string>
      </property>
      <property name="text">
       <string>We couldn't find any properties </string>
      </property>
     </widget>
     <widget class="QFrame" name="frame_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>741</width>
        <height>201</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame{
  border-radius: 25px;
  border: 2px solid white ;
background-color:white;

}
QPushButton{
  border-radius: 10px;
  background: #699CCC;
color:white;



}
QPushButton:hover{
background: #CCD0CD;
color:Black;
}

#pushButton_2{
background: #FF7F7F;
}
#pushButton_2:hover{
background: #CCD0CD;
color:Black;
}
QLabel{
font-size:15px;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>151</width>
         <height>151</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">#label_2{color:black;}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="Resource.qrc">:/Assets/home128.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>40</y>
         <width>211</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Location :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>80</y>
         <width>201</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Space :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_25">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>120</y>
         <width>211</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Bedrooms :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_26">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>40</y>
         <width>231</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Price : </string>
       </property>
      </widget>
      <widget class="QLabel" name="label_27">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>80</y>
         <width>211</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Type :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_28">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>110</y>
         <width>331</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Building Number: 17  </string>
       </property>
      </widget>
      <widget class="QLabel" name="label_29">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>160</y>
         <width>221</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>User ID :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_30">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>160</y>
         <width>211</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>User Name :</string>
       </property>
      </widget>
      <widget class="QToolButton" name="toolButton">
       <property name="geometry">
        <rect>
         <x>680</x>
         <y>20</y>
         <width>31</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QToolButton{
background:white;
border:none;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="Resource.qrc">
         <normaloff>:/Assets/more32.png</normaloff>:/Assets/more32.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
      <widget class="QLabel" name="label_46">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>10</y>
         <width>231</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Property ID : </string>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="frame_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>230</y>
        <width>741</width>
        <height>191</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame{
  border-radius: 25px;
  border: 2px solid white ;
background-color:white;

}
QPushButton{
  border-radius: 10px;
  background: #699CCC;
color:white;



}
QPushButton:hover{
background: #CCD0CD;
color:Black;
}

#pushButton_2{
background: #FF7F7F;
}
#pushButton_2:hover{
background: #CCD0CD;
color:Black;
}
QLabel{
font-size:15px;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <widget class="QLabel" name="label_31">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>151</width>
         <height>161</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">#label_2{color:black;}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="Resource.qrc">:/Assets/home128.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_33">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>30</y>
         <width>211</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Location :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_34">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>70</y>
         <width>201</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Space :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_35">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>110</y>
         <width>211</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Bedrooms :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_36">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>30</y>
         <width>231</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Price : </string>
       </property>
      </widget>
      <widget class="QLabel" name="label_37">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>70</y>
         <width>211</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Type :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_40">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>110</y>
         <width>191</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Building Number:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_41">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>150</y>
         <width>221</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>User ID :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_42">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>150</y>
         <width>211</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>User Name :</string>
       </property>
      </widget>
      <widget class="QToolButton" name="toolButton_2">
       <property name="geometry">
        <rect>
         <x>680</x>
         <y>20</y>
         <width>31</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QToolButton{
background:white;
border:none;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="Resource.qrc">
         <normaloff>:/Assets/more32.png</normaloff>:/Assets/more32.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_7">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>-10</y>
         <width>75</width>
         <height>61</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color:transparent;
border:none;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="Resource.qrc">
         <normaloff>:/Assets/bookmarkv2.png</normaloff>:/Assets/bookmarkv2.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label_45">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>100</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#label_11{
color:black;
  border-radius: 25px;


font-size:15px;
}</string>
    </property>
    <property name="text">
     <string>All Properties :</string>
    </property>
   </widget>
   <zorder>line_8</zorder>
   <zorder>scrollArea_2</zorder>
   <zorder>label</zorder>
   <zorder>label_45</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Resource.qrc"/>
 </resources>
 <connections/>
</ui>
